samza-core/src/main/java/org/apache/samza/config/TaskConfigJava.java

```diff
@@ -117,4 +121,14 @@ public TaskConfigJava(Config config) {
 
     return Collections.unmodifiableSet(allInputSS);
   }
+
+  /**
+   * Returns a value indicating how long to wait for the tasks to shutdown
+   *
+   * @return value indicating how long to wait for the tasks to shutdown
+   */
+  public long getShutdownMs() {
+    if (get(TASK_SHUTDOWN_MS) == null)  return DEFAULT_TASK_SHUTDOWN_MS;
+    return Long.valueOf(get(TASK_SHUTDOWN_MS));
```

Comment:
Let's say if get(TASK_SHUTDOWN_MS) returns some value which cannot be parsed to long, it will throw an exception (NumberFormatException).